loopchain/peer/block_manager.py

```diff
@@ -648,6 +648,10 @@ def vote_unconfirmed_block(self, block: Block, round_: int, is_validated):
             reps_hash=target_reps_hash
         )
 
+        if (self.__channel_service.inner_service.need_to_update_properties(reject_create_tx=None)
```

Comment:
I would like to be able to do it as a standalone function by adding a timer, since additional functions may be needed in the future as well.